src/UpdateTracker.js

```diff
@@ -57,11 +60,17 @@ function trackResource(url) {
   if (!webSocket) {
     const socketUrl = `${protocol.replace('http', 'ws')}//${host}/`;
     webSockets[host] = webSocket = new WebSocket(socketUrl);
+    const backOffDelay = delay || 1000;
     Object.assign(webSocket, { enqueue, onmessage,
       ready: new Promise(resolve => (webSocket.onopen = resolve)),
+      onclose: oncloseFor(host),
+      resources: [],
```

Comment:
Might want to make `resources` a `Set` to avoid multiple re-subscriptions.

Reply:
Done.